commonManual/asciidoc/client-applications.adoc

```diff
@@ -1075,6 +1075,85 @@ The Kerberos authentication token can only be understood by the server if the se
 
 # end::authentication-kerberos-note[]
 
+[role=enterprise-edition]
+[[driver-authentication-bearer]]
+=== Bearer authentication
+
+# tag::authentication-bearer[]
+
+The bearer scheme allows you to create a bearer authentication token using a base64 encoded bearer authentication token provided by an Identity Provider.
+This scheme is used in conjunction with Neo4j's Single Sign-On feature.
+
+[NOTE]
+--
+The bearer scheme requires configuring Single Sign-On on the server.
+See the Operations Manual for information on how to configure it xref:4.4@operations-manual:ROOT:authentication-authorization/sso-integration/index.adoc[Operations Manual -> Integration with Single Sign-On services^].
+Once configured, clients can discover Neo4j's configuration through the Discovery API.
+--
+
+The best way to create a bearer authentication token is shown below:
+
+# end::authentication-bearer[]
+
+[.tabbed-example]
```

Comment:
Pretty sure you can just remove the whole tabbed example as those are not used since we split the manual into the individual languages.

Reply:
I removed it, and then I built the manual with success, everything worked fine.